README.md

```diff
@@ -318,6 +334,12 @@ Here is just a quick taste of what RocketPy is able to calculate. There are hund
 
 ![6-DOF Trajectory Plot](https://raw.githubusercontent.com/RocketPy-Team/RocketPy/master/docs/static/rocketpy_example_trajectory.svg)
 
+If you want to see the trajectory on Google Earth, RocketPy acn easily export a KML file for you:
+
+```python
+test_flight.export_kml(file_name="test_flight.kml")
```

Comment:
I will give a candy to whoever adds a good picture of trajectory plotted on Google Earth

extra candy if it is a GIF